examples/scenes/src/test_scenes.rs

```diff
@@ -53,6 +61,7 @@ macro_rules! scene {
 }
 
 export_scenes!(
+    #[cfg(feature = "cosmic_text")] cosmic_text_scene(cosmic_text),
```

Comment:
This can use:
```suggestion
    #[cfg(feature = "cosmic_text")] cosmic_text_scene(crate::cosmic_text_scene::cosmic_text(), "cosmic text", false),
```